Rakefile

```diff
@@ -26,12 +26,14 @@ Rake::TestTask.new do |t|
 
   case gem_file
   when 'unit.gemfile'
-    t.test_files = FileList['test/api/*_test.rb'] +
-                   FileList['test/solarwinds_apm/*_test.rb'] +
-                   FileList['test/opentelemetry/*_test.rb'] +
-                   FileList['test/noop/*_test.rb'] +
-                   FileList['test/ext/*_test.rb'] +
-                   FileList['test/support/*_test.rb']
+    # t.test_files = FileList['test/api/*_test.rb'] +
+    #                FileList['test/solarwinds_apm/*_test.rb'] +
+    #                FileList['test/opentelemetry/*_test.rb'] +
+    #                FileList['test/noop/*_test.rb'] +
+    #                FileList['test/ext/*_test.rb'] +
+    #                FileList['test/support/*_test.rb'] +
+    #                FileList['test/sampling/*_test.rb']
+    t.test_files = FileList['test/sampling/*_test.rb']
```

Comment:
Is this to stop calling tests for the code that uses the c-extension/legacy stuff? If so, and iiuc what each is testing, we should still test the SW API and other functionality that should still be available on main branch.

Reply:
Updated!